packages/expect/src/__tests__/__snapshots__/toThrowMatchers.test.js.snap

```diff
@@ -383,17 +393,27 @@ Received message: <red>\\"apple\\"</>
 exports[`toThrowError error class threw, but class did not match (non-error falsey) 1`] = `
 "<dim>expect(</><red>received</><dim>).</>toThrowError<dim>(</><green>expected</><dim>)</>
 
-Expected name: <green>\\"Err2\\"</>
+Expected constructor: <green>Err2</>
 
 Received value: <red>undefined</>
 "
 `;
 
+exports[`toThrowError error class threw, but class should not match (error subclass) 1`] = `
+"<dim>expect(</><red>received</><dim>).</>not<dim>.</>toThrowError<dim>(</><green>expected</><dim>)</>
+
+Expected constructor: not <green>Err</>
+Received constructor:     <red>SubErr</>
```

Comment:
Maybe we should print something for subclasses? It might not always be obvious why this failed if the names look unrelated

Reply:
Yeah, it isn’t obvious. Your thoughts how to make it clear are welcome. It reminds me of (positive and negative) assertions for `<` and so on, where report shows the comparison operator.

Reply:
What do you think of this?

<img width="500" alt="not_toBeInstanceOf_base_class" src="https://user-images.githubusercontent.com/11862657/56668033-dc544980-667c-11e9-968c-3828bd03e85b.png">

Reply:
Reminds of the joke:

I’m glad I don’t like X, because if I liked it, I would have to eat it, but I don’t like it!

Reply:
> What do you think of this?

The idea is super awesome, unfortunately it's not quite correct though, `C.prototype instanceof B` would be but that's clunky.
What do you think about `C extends B`?

Reply:
Yes, your suggestion is very intuitive!

Do we extend the notation (pardon pun ;) for `C extends MiddleClass extends B`

Reply:
I guess if it's not too much work, why not. Are there many hundred prototypes long chains in the real world? If so, we might need to limit it?

Reply:
Here is a compromise that might balance clear and correct without over-explaining:

```code
Expected constructor: not Parent
Received constructor:     Child extends Parent
```

```code
Expected constructor: not Ancestor
Received constructor:     Descendant extends … extends Ancestor
```

Reply:
`extends ... extends` is exactly what I thought of 👌